slides/index.html

```diff
@@ -247,20 +261,14 @@ <h1 style="color:DeepSkyBlue">Docker</h1>
         <section>
           <h3>Docker Environment</h3>
           <pre><code class="bash" data-trim data-noescape>
-$ alias de="env | grep DOCKER_"
```

Comment:
There isn't a nice one-liner for alias in powershell, and we only use this once below so I remove it and replaced it below with the full command